source/slang/syntax.h

```diff
@@ -1085,6 +1109,27 @@ namespace Slang
         return FilteredMemberRefList<T>(declRef.getDecl()->Members, declRef.substitutions);
     }
 
+    inline ExtensionDecl* GetCandidateExtensions(DeclRef<AggTypeDecl> const& declRef)
+    {
+        return declRef.getDecl()->candidateExtensions;
+    }
+
+    template<typename T>
+    inline FilteredMemberRefList<T> getMembersOfTypeWithExt(DeclRef<ContainerDecl> const& declRef)
```

Comment:
This is dengerous, because it doesn’t consider whether a *candidate* extension is *actually* applicable to the given specialized declaration.